orb-ui/src/engine/diamond.rs

```
@@ -232,7 +232,7 @@ impl EventHandler for Runner<DIAMOND_RING_LED_COUNT, DIAMOND_CENTER_LED_COUNT> {
             Event::BootComplete { api_mode } => {
                 self.sound
                     .queue(sound::Type::Melody(sound::Melody::BootUp), None)?;
-                self.operator_pulse.stop();
+                let _ = self.operator_pulse.stop(Transition::PlayOnce);
```

Add comment explaining why we discard error.